examples/fabric/reinforcement_learning/train_fabric.py

```diff
@@ -0,0 +1,369 @@
+"""
+Proximal Policy Optimization (PPO) - Accelerated with Lightning Fabric
+
+Adapted from https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo.py
```

Comment:
Let's give credit to you as author :))

```suggestion
Proximal Policy Optimization (PPO) - Accelerated with Lightning Fabric

Author: Federico Belotti @belerico 
Adapted from: https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo.py
```